api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -76,6 +76,15 @@ type OpenStackDataPlaneServiceSpec struct {
 	// OpenStackAnsibleEERunnerImage image to use as the ansibleEE runner image
 	// +kubebuilder:validation:Optional
 	OpenStackAnsibleEERunnerImage string `json:"openStackAnsibleEERunnerImage,omitempty" yaml:"openStackAnsibleEERunnerImage,omitempty"`
+
+	// AnsibleMaxFailPercentage is used to tune service specific, allowable failure percentages during the Ansible execution
+	// https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_error_handling.html#setting-a-maximum-failure-percentage
+	// +kubebuilder:validation:Optional
+	AnsibleMaxFailPercentage int64 `json:"ansibleMaxFailPercentage,omitempty"`
+
+	// AnsibleAnyErrorsFatal is used to tune service specific, any_errors_fatal
+	// +kubebuilder:validation:Optional
+	AnsibleAnyErrorsFatal bool `json:"ansibleAnyErrorsFatal,omitempty"`
```

Comment:
This would apply to all services though right? I guess it depends what we're targeting. My initial understanding of the requirement was that we want the ability to set these on a per-service basis. So that we could have different settings for bootstrap compared to telemetry for example.

Reply:
yes, it would apply to all services (I think it would be useful in some cases),  but now I agree with https://github.com/openstack-k8s-operators/dataplane-operator/pull/519#discussion_r1396042110

Reply:
Ok, it will work that way anyway. So if you just want to do it globally to all services, you could just provide the values under the `ansibleVars`. If you want to be more granular about it on a per service basis, then you could set these values on the service itself.